### PR TITLE
Shell script for getting tpcds count

### PR DESCRIPTION
**Pull request by nkalband**
Added shell script which will get the tpcds tables record counts and compare it with stored std expected counts.
Updated README with same info

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,18 @@ GC_THREADS=9
   ```
   
     Note: Don't leave any hyphen/special characters for db_name.
-
-6. There are 2 types of tpcds benchmark script provided,
+	
+4. To verify the TPCDS database creation and row count of tables with expected standard counts for DB. Use the script provided in the utils directory.
+    It can be used if you have chosen DB size - 1GB/1TB/2TB/3TB/10TB/30TB/100TB.
```
Comment (josiahsams) [inline README.md]: @nkalband , 1 GB Datasize was earlier causing issues. 
10 GB, 100GB, 200GB datasizes are quite commonly used in our environment. 
Can we get the table counts for those sizes too ?

File: README.md [hunk 2]
```diff
@@ -68,8 +68,18 @@ GC_THREADS=9
...
+  
+  ```
+   get_tpcds_count.sh <db_name>
```
Comment (josiahsams) [inline README.md]: Accept DB sizes like 10GB/10gb.. rather than db_name as we don't follow file naming conversions.

File: utils/tpcds_std_rec [hunk 3]
```diff
@@ -0,0 +1,169 @@
+tpcds1gb	call_center	6
```
Comment (josiahsams) [inline utils/tpcds_std_rec]: refer to the previous comment

File: README.md (hunk 1 above)
Comment (nkalband) [reply README.md]: @josiahsams -
I have collected counts for 1TB and 2TB only , rest I updated as per number I got from Kavana.
To get count for  these 3 sizes, I will have to create  these tpcds DB first else If you can provide numbers, I can add them.

Comment (josiahsams) [reply README.md]: @nkalband , Since these are small size DB's, kindly create those DB and update the table row count.

File: README.md (hunk 2 above)
Comment (nkalband) [reply README.md]: @josiahsams - Changes made to take dbsize also argument and using same to store the standard numbers.
Dbname will be required to run queries for existing  DB, so keeping it one of param.

File: README.md (hunk 1 above)
Comment (nkalband) [reply README.md]: @josiahsams  - Updated record data for 10G, 100G and 200G

**Pull request merged**